.gitignore

```diff
@@ -2,6 +2,8 @@
 /js/impress.min.js
 /node_modules
 /npm-debug.log
+/DS_Store
+/examples/DS_Store
 /*.tgz
```

Comment:
Please submit these in an independent PR.

Also, if you just add a single

    DS_Store

That will cover all possible files called DS_Store. (I think?)